### PR TITLE
Npm scripts

### PR DESCRIPTION
**Pull request by mattyod**
# Description

Various script updates for pipeline.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,13 +9,12 @@
     "start": "NODE_PATH=. node server.js",
     "setup": "NODE_PATH=.",
     "dev": "NODE_PATH=. NODE_ENV=development node-dev server.js",
-    "test": "NODE_PATH=. NODE_ENV=testing nyc --reporter=text mocha 'test/unit/**/*.js'",
-    "test:unit": "NODE_PATH=. NODE_ENV=testing nyc --reporter=text mocha 'test/unit/**/*.js'",
+    "test": "NODE_PATH=. NODE_ENV=testing mocha 'test/unit/**/*.js'",
```
Comment (dtaylor7) [inline package.json]: how come you no longer specify reporter and not using nyc?

File: package.json [hunk 2]
```diff
@@ -9,13 +9,12 @@
     "start": "NODE_PATH=. node server.js",
     "setup": "NODE_PATH=.",
     "dev": "NODE_PATH=. NODE_ENV=development node-dev server.js",
-    "test": "NODE_PATH=. NODE_ENV=testing nyc --reporter=text mocha 'test/unit/**/*.js'",
-    "test:unit": "NODE_PATH=. NODE_ENV=testing nyc --reporter=text mocha 'test/unit/**/*.js'",
```
Comment (dtaylor7) [inline package.json]: this is required for cnp pipeline...

File: package.json [hunk 3]
```diff
@@ -9,13 +9,12 @@
...
+    "test": "NODE_PATH=. NODE_ENV=testing mocha 'test/unit/**/*.js'",
     "test:e2e": "NODE_PATH=. NODE_ENV=testing LOG_LEVEL=OFF codeceptjs run -c test/e2e/codecept.conf.js --steps --invert",
-    "test:coverage": "NODE_PATH=. NODE_ENV=testing LOG_LEVEL=OFF istanbul cover _mocha 'test/unit/**/*.js' -- --recursive --timeout 10000",
-    "posttest:coverage": "istanbul check-coverage --statements 80 --branches 80 --functions 80 --lines 80",
```
Comment (dtaylor7) [inline package.json]: how come these have changed?

File: package.json (hunk 2 above)
Comment (mattyod) [reply package.json]: test is, test:unit isn't

File: package.json (hunk 3 above)
Comment (mattyod) [reply package.json]: because current coverage level is less than 80. Looking to fix that in another commit.

**Pull request merged**